### PR TITLE
NFT Event Test Blog Post with streaming video embed

### PR DESCRIPTION
**Pull request by smombartz**
# Description
created a blog post page that could potentially house livestream of NFT event. the video embed is a placeholder i can create a new one from unlock youtube account.

we would still need a way to embed the lock js script. it this a possible option: https://stackoverflow.com/questions/2754391/embed-javascript-in-markdown/7219523

# Screenshot

screencapture-localhost-3002-blog-nft-dev-meetup-2019-05-07-18_31_33

### CODE REVIEW

File: unlock-protocol.com/blog/nft-dev-meetup.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+title: NFT Dev Meetup
+subTitle: Livestream of the NFT Dev Meetup
+authorName: Sascha Mombartz
+publishDate: May 16, 2019
+description: More info and a livestream for registred participants.
+image: /static/images/blog/nftdevmeetup/hero.jpg
+---
+![Optimistic Unlocking in action.](/static/images/blog/nftdevmeetup/hero.jpg)
+
+We’re doing something different! By now you’ve probably been to a few too many after-parties. Let's not even talk about Consensys - did you see how packed the hallways were?
```
Comment (benwerd) [inline unlock-protocol.com/blog/nft-dev-meetup.md]:
```suggestion
We’re doing something different! By now you’ve probably been to a few too many after-parties. Let's not even talk about Consensus - did you see how packed the hallways were?
```

This should be Consensus not Consensys: a typo fix on this elsewhere was erroneous and will be changed back.

**Comment by benwerd** [comment]
The paywall code has now been merged. @julien51, do you want to revisit your review?

**Comment by smombartz** [comment]
@julien51 what do you think about changing this blogpost title to "One Key, Multi Access" and i can describe the concept of how you can use locks across multiple properties.

**Comment by cellog** [comment]
> @julien51 what do you think about changing this blogpost title to "One Key, Multi Access" and i can describe the concept of how you can use locks across multiple properties.

This sounds like a neat idea. Let's talk about it this week?

**Comment by smombartz** [comment]
i've created a blog file with images. for edits please make suggestions and comments in [gdoc](https://docs.google.com/document/d/1Q84E5xfnqIpijh3NuINFElVJF49V968SpLEJ-VrJcJY).

screencapture-localhost-3002-blog-multi-use-locks-2019-05-28-12_43_12

**Pull request merged**